base/distributed/managers.jl

```diff
@@ -470,7 +470,7 @@ function socket_reuse_port()
             # provide a clean new socket
             return TCPSocket()
         end
-        is_apple() && bind_client_port(s)
+        is_bsd() && bind_client_port(s)
```

Comment:
Should change this to `!is_linux() && bind_client_port(s)`. 

Or ideally, few lines above to `bind_port = is_linux() ? true : false ; bind_port && bind_client_port(s)` and this line to `!bind_port && bind_client_port(s)`.